conf/gaurun.toml

```diff
@@ -13,6 +13,7 @@ timeout = 5 # sec
 keepalive_timeout = 30
 keepalive_conns = 4
 retry_max = 1
+use_fcm = true
```

Comment:
In **conf/gaurun.toml**, each value is set default value except for the part. So, set false.